client/src/components/Onboarding/OnboardingAdmin.js

```diff
@@ -132,37 +134,41 @@ const OnboardingAdminListUser = ({ user, viewLogCallback }) => {
   const adminAction = useSelector(state => state.onboarding.action);
 
   return (
+    /* !!!: Temporary extra markup to make simpler PR diff */
+    /* eslint-disable prettier/prettier */
     <tr styleName="user">
       <td styleName="name">
-        <div>{`${user.firstName} ${user.lastName}`}</div>
+        {`${user.firstName} ${user.lastName}`}
       </td>
-      <td>
+      <td colSpan={stepTableColumnCount}>
+        <table styleName="steps">
+          <thead>
+            <tr>
+              <th>Step</th>
+              <th>Status</th>
+              <th colSpan="2">Administrative Actions</th>
+              <th>Log</th>
+            </tr>
+          </thead>
+          <tbody>
```

Comment:
This appears misaligned when multiple users are present. To test: uncomment line `#259` in `onboarding/api/views.py`
<img width="1526" alt="Screen Shot 2021-03-16 at 10 22 40 AM" src="https://user-images.githubusercontent.com/20326896/111334788-dc2f0f80-8641-11eb-917c-63acdde2e0cf.png">

Reply:
Thank you.